bc_obps/registration/schema/user_operator.py

```diff
@@ -112,6 +102,13 @@ def resolve_partner_operators_array(obj: UserOperator) -> Optional[List[PartnerO
             ]
         return None
 
+    @staticmethod
+    def resolve_role(obj: UserOperator) -> str:
+        role = obj.role
+        if role == UserOperator.Roles.ADMIN:
+            return 'Administrator'
+        return obj.role.capitalize()
```

Comment:
This seemed easier than manipulating the formData on the front end or building a special widget to convert the roles into the text we want to see